docs/graphql/manual/api-reference/schema-metadata-api/scheduled-triggers.rst

```diff
@@ -0,0 +1,294 @@
+.. meta::
+   :description: Manage scheduled triggers with the Hasura schema/metadata API
+   :keywords: hasura, docs, schema/metadata API, API reference, scheduled trigger
+
+Schema/Metadata API Reference: Scheduled Triggers
+=================================================
+
+.. contents:: Table of contents
+  :backlinks: none
+  :depth: 1
+  :local:
+
+Scheduled triggers are used to invoke webhooks based on a timestamp or cron.
+
+.. _create_cron_trigger:
+
+create_cron_trigger
+-------------------
+
+``create_cron_trigger`` is used to create a new cron trigger.
+
+.. code-block:: http
+
+   POST /v1/query HTTP/1.1
+   Content-Type: application/json
+   X-Hasura-Role: admin
+
+   {
+       "type" : "create_cron_trigger",
+       "args" : {
+           "name": "sample_cron",
+           "webhook": "https://httpbin.org/post",
+           "schedule":  "* * * * *",
+           "payload": {
+               "key1": "value1",
+               "key2": "value2"
+           },
+           "include_in_metadata":false,
+           "comment":"sample_cron commment"
+       }
+   }
+
+.. _create_cron_trigger_syntax:
+
+Args syntax
+^^^^^^^^^^^
+
+.. list-table::
+   :header-rows: 1
+
+   * - Key
+     - Required
+     - Schema
+     - Description
+   * - name
+     - true
+     - TriggerName_
+     - Name of the cron trigger
+   * - webhook
+     - true
+     - :ref:`WebhookURL <WebhookURL>`
+     - URL of the webhook
+   * - schedule
+     - true
+     - Cron Expression
+     - Cron expression at which the trigger should be invoked.
+   * - payload
+     - false
+     - JSON
+     - Any JSON payload which will be sent when the webhook is invoked.
+   * - headers
+     - false
+     - [ HeaderFromValue_ | HeaderFromEnv_ ]
+     - List of headers to be sent with the webhook
+   * - retry_conf
+     - false
+     - RetryConfST_
+     - Retry configuration if scheduled invocation delivery fails
+   * - include_in_metadata
+     - false
+     - Boolean
+     - Flag to indicate whether a trigger should be included in the metadata. When a trigger is included in the metadata, the user will be able to export it when the metadata of the graphql-engine is
+       exported.
+   * - comment
+     - false
+     - Text
+     - Custom comment.
+   * - replace
+     - false
+     - Bool
+     - When replace is set to ``true``, the cron trigger will be updated(if exists) and when it's ``false`` or the
```

Comment:
```suggestion
     - When replace is set to ``true``, the cron trigger will be updated (if it exists) and if it's ``false``. If the
```

Reply:
Here I'm a bit confused. I thought the cron trigger will be updated when replace is set to true? Why do we later say "if it's false"?

Reply:
I'm explaining the behaviour of the API when replace is `false`.